x/stake/keeper/key.go

```diff
@@ -154,6 +157,12 @@ func GetUBDsByValIndexKey(valAddr sdk.ValAddress) []byte {
 	return append(UnbondingDelegationByValIndexKey, valAddr.Bytes()...)
 }
 
+// gets the prefix for all unbonding delegations from a delegator
+func GetUnbondingDelegationTimeKey(timestamp time.Time) []byte {
+	bz, _ := timestamp.MarshalBinary()
```

Comment:
Do you think we should panic here and in `GetRedelegationTimeKey` to be defensive/safe?